pygeos/geometry.py

```diff
@@ -537,3 +539,81 @@ def get_num_geometries(geometry):
     0
     """
     return lib.get_num_geometries(geometry)
+
+
+@requires_geos("3.6.0")
+@multithreading_enabled
+def get_precision(geometry):
+    """Get the precision of a geometry.
+
+    If a precision has not been previously set, it will be 0 (floating point precision).
+    Otherwise, it will return the grid size that was set on a geometry.
+
+    Returns NaN for not-a-geometry values.
+
+    Parameters
+    ----------
+    geometry : Geometry or array_like
+
+    See also
+    --------
+    set_precision
+
+    Examples
+    --------
+    >>> get_precision(Geometry("POINT (1 1)")) == 0
+    True
+    >>> geometry = set_precision(Geometry("POINT (1 1)"), 1)
+    >>> get_precision(geometry) == 1
+    True
+    >>> np.isnan(get_precision(None))
+    True
+    """
+    return lib.get_precision(geometry)
+
+
+@requires_geos("3.6.0")
+@multithreading_enabled
+def set_precision(geometry, grid_size=0, preserve_topology=False):
```

Comment:
I don't think it make sense to provide `0` as a default, because nothing will really happen if you do `grid_size=0`.

Reply:
Removed default value, now will be required parameter.